### PR TITLE
Service-Broker-Setup: Use values files instead of command line

### PR DESCRIPTION
**Pull request by mook-as**
This looks more like the SCF deployment mechanism, but there is less
interpolation and more manual editing.

### CODE REVIEW

File: Service-Broker-Setup.md [hunk 1]
```diff
@@ -13,50 +13,66 @@ a CAP cluster.
...
+You need an external MySQL installation, with account credentials that allow creating and deleting both databases and users.
+
+## Configuring the deployment
+
+Create a values.yaml file (the rest of the document assumes it is called `usb-config-values.yaml`) with the settings required for the install.  Use the file below as a template, and modify the values to suit your installation.
+
+```yaml
+env:
+  # Database access credentials; the given user must have privileges to create
+  # delete both databases and users
```
Comment (andreas-kupries) [inline Service-Broker-Setup.md]: ...create __and__ delete... . The `and` is missing in the sentence.

Comment (mook-as) [reply Service-Broker-Setup.md]: Fixed in 9bdb11490e74fdbd3a616b863807e1bb7934077b

File: Service-Broker-Setup.md [hunk 2]
```diff
@@ -13,50 +16,69 @@ a CAP cluster.
...
+  # UAA_CA_CERT can be obtained with the command line:
+  #   kubectl get secret -n $NAMESPACE secret-$REVISION -o jsonpath='{.data.uaa-ca-cert}' | base64 -d
+  UAA_CA_CERT:|
+    -----BEGIN CERTIFICATE-----
+    MIIETm8gcmVhbGx5IEkgc2FpZCB0aGVyZSBpcyBubyBzZWNyZXQgbWVzc2FnZSEhCg==
+    -----END CERTIFICATE-----
+
+  SERVICE_TYPE: mysql # Optional
+
+# The whole "kube" section is optional
```
Comment (troytop) [inline Service-Broker-Setup.md:68]: Omit

File: Service-Broker-Setup.md [hunk 3]
```diff
@@ -13,50 +16,69 @@ a CAP cluster.
...
-    | `CLUSTER_ADMIN_PASSWORD` | The administrator password for the CAP deployment | `CLUSTER_ADMIN_PASSWORD="$(kubectl get secret --namespace $NAMESPACE secret-$REVISION -o jsonpath='{.data.cluster-admin-password}' \| base64 -d)"`
-    | `DOMAIN` | The domain that CAP is using for deployed applications | `DOMAIN="$(kubectl get pod --namespace $NAMESPACE api-0 -o jsonpath='{.spec.containers[0].env[?(@.name == "DOMAIN")].value}')"`
-    | `CF_CA_CERT` | The certificate for the internal certificate authority for the CAP deployment | `CF_CA_CERT="$(kubectl get secret --namespace $NAMESPACE secret-$REVISION -o jsonpath='{.data.internal-ca-cert}' \| base64 -d)"`
-    | `UAA_CA_CERT` | The certificate for the internal certificate authority for the UAA deployment | `UAA_CA_CERT="$(kubectl get secret --namespace $NAMESPACE secret-$REVISION -o jsonpath='{.data.uaa-ca-cert}' \| base64 -d)"`
-    | `SIDECAR_NAMESPACE` | The Kubernetes to install the sidecar to; may be the same as the CAP namespace | `SIDECAR_NAMESPACE=${NAMESPACE}`
+When deploying the chart, a Kubernetes namespace to install the sidecar to is
```
Comment (troytop) [inline Service-Broker-Setup.md:79]: What is a sidecar (as far as the user knows)?

**Review by troytop — APPROVED** [review]
Some small things could be changed, but that can happen later. The process is workable with these steps.

**Pull request merged**